### PR TITLE
fix(artwork-brick): Fix artwork hover zoom performance

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Fix**

### Description

Perf on this has become janky over the past few months. This fixes that: 

https://github.com/user-attachments/assets/6449cbcf-8ac6-47fb-9062-cb48ab037984

cc @artsy/diamond-devs

### CODE REVIEW

File: src/Components/MagnifyImage.tsx [hunk 1]
```diff
@@ -63,9 +55,8 @@ export const MagnifyImage: React.FC<React.PropsWithChildren<MagnifyImageProps>>
     const x = e.clientX - containerRect.left
     const y = e.clientY - containerRect.top
 
-    setPositions({
-      x,
-      y,
+    requestAnimationFrame(() => {
```
Comment (damassi) [inline src/Components/MagnifyImage.tsx:58]: ☝️ the fix

**Review by egdbear — APPROVED** [review]

File: src/Components/MagnifyImage.tsx (hunk 1 above)
Comment (starsirius) [reply src/Components/MagnifyImage.tsx:58]: Nice! So I guess without `requestAnimationFrame`, it'd be rendering faster than it needs to be and that caused the possible janky animation.

File: src/Components/MagnifyImage.tsx [hunk 2]
```diff
@@ -78,16 +69,16 @@ export const MagnifyImage: React.FC<React.PropsWithChildren<MagnifyImageProps>>
...
-      ref={containerRef as any}
+      ref={containerRef}
     >
       <Image
         {...rest}
         title={undefined} // don't display a tooltip when hovering over the image
         style={{
           ...rest.style,
-          transformOrigin: getTransformOrigin(positions),
-          transition: `transform ${scaleDuration}s ease, transform-origin 100ms ease, opacity 0.25s`,
```
Comment (starsirius) [inline src/Components/MagnifyImage.tsx:89]: TIL `transform-origin` is invalid here!

**Comment by dzucconi** [comment]
Future optimizations could include disabling this behavior during scroll and eliminating state management entirely in favor of direct DOM manipulation.

**Pull request merged**